Makefile.ibi

```diff
@@ -62,15 +51,15 @@ openshift-installer:
 	fi
 
 .PHONY: ibi-iso
-ibi-iso: $(IB_CLI) $(SSH_KEY_PRIV_PATH) credentials/pull-secret.json image-based-install-iso.yaml ## Create ISO to be used in IBI
+ibi-iso: $(SSH_KEY_PRIV_PATH) credentials/pull-secret.json image-based-installation-config.yaml ## Create ISO to be used in IBI
 	mkdir -p $(IBI_WORK_DIR)
-	$(IB_CLI) create-iso --dir $(IBI_WORK_DIR)
+	OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE=$(DEFAULT_RELEASE_IMAGE) $(OPENSHIFT_INSTALLER_BIN) image-based create image --dir $(IBI_WORK_DIR)
```

Comment:
If the previous change is accepted, then here we should change `DEFAULT_RELEASE_IMAGE` to `IBI_RELEASE_IMAGE`